javascript/ql/src/experimental/Security/CWE-347-noVerification/JsonWebToken.ql

```diff
@@ -0,0 +1,65 @@
+/**
+ * @name JWT missing secret or public key verification
+ * @description The application does not verify the JWT payload with a cryptographic secret or public key.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 8.0
+ * @precision high
+ * @id js/jwt-missing-verification-jsonwebtoken
```

Comment:
You're using `js/jwt-missing-verification-jsonwebtoken` in 3 different `.ql` files. 
Each `.ql` file should have a unique `@id`.